rts/Lua/LuaMatrix.hpp

```diff
@@ -0,0 +1,197 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
+#ifndef LUA_MATRIX_H
+#define LUA_MATRIX_H
```

Comment:
Filename and ifndef should match, and also our headers should end in .h instead of .hpp for consistency with the rest of the project